truss/config/trt_llm.py

```diff
@@ -54,7 +54,7 @@ class TrussTRTLLMBuildConfiguration(BaseModel):
     max_input_len: int
     max_output_len: int
     max_batch_size: int
-    max_beam_width: int
+    max_beam_width: int = 1
```

Comment:
if this is always 1, why even expose it as an option for a user to change? will just cause confusion.

Reply:
I don't think we should remove part of the interface because we aren't able to leverage it in the runtime, my thinking is that we eventually will support this. Maybe the more explicit fix is to add a validator to ensure that this is set to 1 for now, and remove it in the future